python/qpid_dispatch_internal/tools/display.py

```diff
@@ -138,6 +143,33 @@ def num(self, value):
     value /= 1000
     return self.numCell(value, 'g')
 
+def PlainNum(value):
+    if value < 10:
+      return "%1d" % value
```

Comment:
You might want to wrap this in a try/except block to catch formatting errors.   Simply return "%s" % value on error.

Try  "%d" % "HI GANESH" in python -> TypeError

Reply:
Will do !